receiver/mysqlreceiver/README.md

```diff
@@ -0,0 +1,43 @@
+# MySQL Receiver
+
+This receiver queries MySQL's global status and InnoDB tables.
+
+Supported pipeline types: `metrics`
+
+> :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
+
+## Prerequisites
+
+This receiver supports MySQL version 8.0
```

Comment:
This should read `5.7+`. I will make the update in the next commit or PR.

Reply:
Does it also support MySQL variants such as MariaDB and Percona?

Reply:
This is a good question. It's not an immediate goal, though I'm sure a reasonable one for future revisions.